docs/faq.md

```diff
@@ -4,27 +4,27 @@
 
 ### Product
 
-**What is the difference between eZ Platform and Enterprise Edition?**
+**What is the difference between eZ Platform and eZ Enterprise?**
 
-eZ Platform is open-source. Enterprise Edition bases on eZ Platform and adds new functionalities.
+eZ Platform is open-source. eZ Enterprise bases on eZ Platform, but features new functionalities.
 
-**Which parts of eZ are open source and which are commercial?**
+**Which parts of eZ are open-source and which are commercial?**
```

Comment:
```suggestion
**Which parts of eZ are open source and which are commercial?**
```
`open source` is more common in our documentation. If you want to change that, let's discuss and add to terminology :)